doc/api/globals.md

```diff
@@ -598,13 +598,30 @@ The object that acts as the namespace for all W3C
 [WebAssembly][webassembly-org] related functionality. See the
 [Mozilla Developer Network][webassembly-mdn] for usage and compatibility.
 
+## `reportError(error)`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+<!-- type=global -->
```

Comment:
I think we should land this as experimental given that it's not obvious which behavior it should follow.
```suggestion
> Stability: 1 - Experimental.

<!-- type=global -->
```